README.md

```diff
@@ -1,6 +1,6 @@
 # Module Documentation
 
-## Module Data.Validation
+## Module Data.Validation.Semigroup
```

Comment:
Can we keep this as is? Otherwise, I have to make a major release and change the book.